robottelo/ui/activationkey.py

```diff
@@ -85,6 +85,37 @@ def search_key(self, element_name):
             element = self.wait_until_element((strategy, value % element_name))
         return element
 
+    def search_key_subscriptions(self, ak_name, subscription_name):
+        """Fetch associated subscriptions from selected activation key"""
+
+        activation_key = self.search_key(ak_name)
+        if activation_key is None:
+            raise UINoSuchElementError(
+                'Could not find activation key {0}'.format(ak_name))
```

Comment:
Prefix the string with `u` because we can have unicode strings as activation key name.